source/reference/s3-api-compatibility.rst

```diff
@@ -151,3 +157,11 @@ Unsupported API Bucket Endpoints
    ListBucketAnalyticsConfigurations
    DeleteBucketAnalyticsConfiguration
    CreateSession
+
+MinIO alternatives for unsupported Bucket resources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- For calls to ``BucketACL`` or ``ObjectACL`` operations, use :ref:`Policies <minio-policy>`.
+- Calls to ``BucketCORS`` operations are not needed because CORS is enabled by default on all buckets for all HTTP verbs.
+- For calls to ``BucketWebsite`` operations, use ``caddy`` or ``nginx``.
```

Comment:
It it worth linking to the caddy and nginx project sites?

Reply:
Felt like can of worms (why only these two), plus no links in original moved